src/main/resources/static/documentation.md

```diff
@@ -0,0 +1,174 @@
+# SonarScanner for Maven
+The SonarScanner for Maven is recommended as the default scanner for Maven projects.
+
+The ability to execute the SonarQube analysis via a regular Maven goal makes it available anywhere Maven is available (developer build, CI server, etc.), without the need to manually download, set up, and maintain a SonarQube scanner installation. The Maven build already has much of the information needed for SonarQube to successfully analyze a project. By preconfiguring the analysis based on that information, the need for manual configuration is reduced significantly.
+
+[Prerequisites](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-maven/#prerequisites "Prerequisites")
+---------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Is there additional value to using the hyphen lines instead of `##` here?
```suggestion
## [Prerequisites](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-maven/#prerequisites "Prerequisites")
```